package.json

```
@@ -11,10 +11,11 @@
     "url": "https://github.com/FuelLabs/sway-vscode-plugin.git"
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -b",
-    "watch": "tsc -b -w",
-    "test": "sh ./scripts/e2e.sh",
+    "vscode:prepublish": "npm run esbuild-base -- --minify",
+    "esbuild-base": "esbuild ./client/src/main.ts --bundle --outfile=client/out/main.js --external:vscode --format=cjs --platform=node",
+    "esbuild": "npm run esbuild-base -- --sourcemap",
+    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
```

It would be convenient to have `npm run package` and update the docs.

```suggestion
    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
    "package": "vsce package -o sway-vscode-plugin-dev.vsix",
```